src/Actions/EmailWorkflowAction.php

```diff
@@ -0,0 +1,138 @@
+        }
+        else if ($this->EmailTarget === 'Field') {
+            $newFields->push(TextField::create('EmailField', $toTitle)->setDescription('Name of target field'));
+        }
+        else if ($this->EmailTarget === 'Member') {
+            $newFields->push(DropdownField::create('EmailMemberID', $toTitle, Member::get()->map()));
+        }
+        else if ($this->EmailTarget === 'Group') {
+            $newFields->push(DropdownField::create('EmailGroupID', $toTitle, Group::get()->map()));
+        }
+
+        $fields->insertBefore('Property', $newFields);
+        $this->extend('updateTargetMethodCMSFields', $fields);
+        return $fields;
+    }
+
+    public function fieldLabels($relations = true)
+    {
+        return array_merge(parent::fieldLabels($relations), [
+            'EmailTarget' => _t('EmailWorkflowAction.EMAILTARGET', 'Target type'),
+            'EmailFrom' => _t('EmailWorkflowAction.EMAILFROM', 'Send from'),
+            'EmailSubject' => _t('EmailWorkflowAction.EMAILSUBJECT', 'Email subject'),
+            'EmailTemplateID' => _t('EmailWorkflowAction.EMAILTEMPLATE', 'Email template'),
+        ]);
+    }
+
+    public function execute(WorkflowInstance $workflow)
+    {
+        parent::execute($workflow);
+
+        $target = $workflow->getTarget();
+        $from = $this->EmailFrom;
+        $subject = $this->EmailSubject;
+        $template = $this->EmailTemplate();
+
+        if (!$target || !$from || !$subject || !$template) {
+            return false;
+        }
+
+        $addys = [];
+        if ($this->EmailTarget === 'Manual') {
+            $addys[] = $this->EmailManual;
+        }
+        else if ($this->EmailTarget === 'Field') {
+            $addys[] = $target->{$this->EmailField};
+        }
+        else if ($this->EmailTarget === 'Member') {
+            if ($member = $this->EmailMember()) {
+                if (filter_var($member->Email, FILTER_VALIDATE_EMAIL)) {
+                    $addys[] = $member->Email;
+                }
+            }
+        }
+        else if ($this->EmailTarget === 'Group') {
+            if ($group = $this->EmailGroup()) {
+                foreach ($group->Members() as $member) {
+                    if (filter_var($member->Email, FILTER_VALIDATE_EMAIL)) {
+                        $addys[] = $member->Email;
+                    }
+                }
+            }
+        }
+
+        try {
+            $body = $target->customise(['FromEmail' => $from])->renderWith($template->getTemplateFile());
+            foreach ($addys as $to) {
+                $email = Email::create();
+                $email->setTo($to);
+                $email->setFrom($from);
+                $email->setSubject($subject);
+                $email->setBody($body->getValue());
+                $email->setPlainTemplate($body->Plain());
+                $email->send();
+            }
+            return true;
+        } catch (Exception $e) {
+            return false;
```

Comment:
And here too - having some kind of email config issue shouldn't block the whole workflow up